Lesson 2/src/style.css

```
@@ -0,0 +1,77 @@
+body {
+  max-width: 940px;
+  margin: 0 auto;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  background-color: #18191c;
+}
+
+ul {
+  display: flex;
+  flex-wrap: nowrap;
+  gap: 20px;
+}
+
+nav li {
+  border-bottom: 2px solid orange;
+}
+
+main {
+  margin: 30px 0;
+}
+
+h1 {
+  margin-bottom: 30px;
+  font-size: 26px;
+}
+
+.wrapper {
+  display: flex;
+  flex-wrap: nowrap;
+  gap: 20px;
+}
+
+.card {
+  display: flex;
+  flex-basis: calc((940px - 40px) / 3);
```

```suggestion
  flex: 0 0 calc((100% - 40px) / 3);
```

лучше использовать 100% т.к. если ширина поменяется, придётся и здесь менять